xrfeitoria/sequence/sequence_unreal.py

```diff
@@ -148,6 +169,7 @@ def spawn_actor_with_keys(
         actor_name: Optional[str] = None,
         stencil_value: int = 1,
         anim_asset_path: 'Optional[str]' = None,
+        motion_data: 'Optional[]' = None,
```

Comment:
```suggestion
        motion_data: 'Optional[List[MotionFrame]]' = None,
```

Reply:
fixed in f93d86b1b32bd33c3591db273416b4e56a000f7d